src/features/Settings/components/GeneralSettings.tsx

```diff
@@ -123,40 +109,24 @@ const GeneralSettings: FC<SettingsProps> = ({ globalSettings, updateGlobalSettin
     <DashboardPanel heading={translate('SETTINGS_GENERAL_LABEL')}>
       <Divider />
       <SettingsField>
-        <SettingsLabel>
-          {translate('SETTINGS_HANDLING_LABEL')}
-          <Tooltip tooltip={<span>{translate('SETTINGS_TOOLTIP')}</span>}>
-            <div>
-              <SettingsTooltipIcon icon="shape" />
-            </div>
-          </Tooltip>
-        </SettingsLabel>
+        <SubHeading fontWeight="initial">
+          {translate('SETTINGS_HANDLING_LABEL')}{' '}
+          <Tooltip tooltip={<span>{translate('SETTINGS_TOOLTIP')}</span>} />
+        </SubHeading>
         <SettingsControl>
           <Link to={ROUTE_PATHS.SETTINGS_IMPORT.path}>
             <SettingsButton secondary={true}>{translate('SETTINGS_IMPORT_LABEL')}</SettingsButton>
           </Link>
-          <Link to={ROUTE_PATHS.SETTINGS_EXPORT.path}>
+          <Link to={ROUTE_PATHS.SETTINGS_EXPORT.path} style={{ marginLeft: SPACING.SM }}>
```

Comment:
Do we want to use `style`?

Reply:
It's seemed preferable over creating a new SComponent, a class or overriding Link.
If the need emerges again we can extend Link with style-system.
The right tool for the job.